app/controller/hockey_site.rb

```diff
@@ -7,7 +7,8 @@ class HockeySite
                  teams: "./data/team_info.csv")
   def self.call(env)
     path = env["PATH_INFO"]
-    @@query = env["QUERY_STRING"].split("=").last
+    # require 'pry'; binding.pry
```

Comment:
Will break a few things in my pull request, so anytime we see @@query we just need to add the aproppriet key.